src/Compilers/Core/MSBuildTaskTests/TestUtilities/EventArgsFormatting.cs

```diff
@@ -0,0 +1,304 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Globalization;
+using System.Collections.Generic;
+using System.Text;
+
+using Microsoft.Build.Framework;
+
+namespace Microsoft.CodeAnalysis.BuildTasks.UnitTests
+{
+    /// <summary>
+    /// Offers a default string format for Error and Warning events
+    /// </summary>
+    internal static class EventArgsFormatting
```

Comment:
Where does this come into play? I couldn't find any reference to `EventArgsFormatting` or `FormatEventMessage` in other modified files.

Reply:
Forgot to delete.